.circleci/config.yml

```diff
@@ -2,16 +2,30 @@ version: 2.1
 jobs:
   test:
     docker:
-      - image: circleci/node:lts
+      - image: dkhamsing/awesome_bot
+    parallelism: 50
     steps:
+      - checkout
       - run:
-          name: Test dependencies
```

Comment:
do we not need the `Test dependencies` job that was here before?

Reply:
It's not necessary, the past job was used as a empty job until we have this pipeline ready.